src/config.rs

```diff
@@ -40,12 +40,14 @@ fn true_bool() -> bool {
 #[derive(Clone, Debug, Deserialize)]
 pub struct Selection {
     pub semantic_escape_chars: String,
+    pub save_to_clipboard: Option<bool>,
```

Comment:
It looks like there's no code path for this to be `None`; a `#[serde(default)]` attribute on this field will solve the config back-compat issue without need for option handling elsewhere in the code.